PatientRegistrationApp/src/components/HouseholdReviewSubmit.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<v-container fluid>
 		<v-row align="center" justify="center">
-			<v-heading> Please review all entries before submitting the registration! </v-heading>
+			<v-heading> Ensure information is correct before submission. </v-heading>
```

Comment:
Looks like there was a little confusion here. You were supposed to get rid of (2) and make (1) read "Please ensure your information is correct" or "Please ensure your household's information is correct" for the household workflow. 

Also, make sure you merge development into your branch before you merge your PR. You're 4 commits behind and I'm pretty sure one or two of those affect this code.

![image](https://user-images.githubusercontent.com/67079030/98501171-11e76600-2214-11eb-92c4-e8ef12572f2d.png)

Reply:
Gotcha that makes more sense. On my end it says there are no conflicts with the development branch. Do you think it would be better to delete this current branch and start over since there's not much to this card?